templates/common/openstack/files/openstack-cacert-file.yaml

```diff
@@ -0,0 +1,6 @@
+filesystem: "root"
```

Comment:
Is there a way to only copy the cert to `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem` when it is set (non empty)?
The clouds.conf shouldn't have the `ca-file` option in this case so we're not breaking anything.